### PR TITLE
Boos 179 swagger ghpages

### PR DESCRIPTION
**Pull request by Chichilele**
* Refactor openapi docs in `docs` folder
* openAPI docs to GitHub pages: https://intellium-ai.github.io/SPA-Flask-React-boilerplate/ (simple HTML file loading Swagger packages)

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<html>
+    <head>
+        <!-- Load the latest Swagger UI code and style from npm using unpkg.com -->
+        <script src="https://unpkg.com/swagger-ui-dist@3/swagger-ui-bundle.js"></script>
+        <link rel="stylesheet" type="text/css" href="https://unpkg.com/swagger-ui-dist@3/swagger-ui.css"/>
+        <title>My New API</title>
```
Comment (koutsilisk) [inline docs/index.html:6]: Intellium boilerplate API or something equivalent

**Review by koutsilisk — CHANGES_REQUESTED** [review]
Change the html title on index.html to reflect the actual API, My new API is not that helpful.

**Comment by Chichilele** [comment]
> Change the html title on index.html to reflect the actual API, My new API is not that helpful.

good point, not sure it shows anywhere though

**Comment by koutsilisk** [comment]
It shows on the tab

**Review by koutsilisk — APPROVED** [review]

**Comment by Chichilele** [comment]
I'll actually merge this and have @joaopcartier 's review done directly on GH pages and reported on the https://intellium-ai.atlassian.net/browse/BOOS-179?atlOrigin=eyJpIjoiNWQyYjE3NzYxMTA1NDY2Nzk2MTc3MTA3ODgyYTBiYWIiLCJwIjoiaiJ9

**Pull request merged**